README.md

```diff
@@ -48,12 +48,17 @@ Pre-built pip and conda packages support Ubuntu 18.04+, macOS 10.15+ and
 Windows 10 (64-bit) with Python 3.6-3.9.
 
 ```bash
-# Install Open3D stable release with pip (including in conda virtual environments)
-$ pip install open3d
+# Install
+pip install open3d
 
```

Comment:
Should we add `python -c "import open3d as o3d; print(o3d.__version__)"` here to keep it consistent with the docs?

Reply:
added